packages/core/src/initialization/make-index-api.ts

```diff
@@ -27,13 +11,26 @@ export function makeIndexApi(
   network: Networks,
   logger: DiagnosticsLogger
 ): DatabaseIndexApi | undefined {
-  if (process.env.CERAMIC_ENABLE_EXPERIMENTAL_COMPOSE_DB != 'true') {
-    return undefined
-  }
   if (!indexingConfig) {
     logger.warn(`Indexing is not configured. Please add the indexing settings to your config file`)
     return undefined
   }
+
+  if (indexingConfig.composedbEnabled === false) {
+    logger.warn(
+      'Indexing is actively disabled. Change the corresponding CLI flag or ENV variable to enable it.'
+    )
+    return undefined
+  }
+
+  // TODO: extend with dynamic config check if MAINNET or sync is enabled
+  if (network === Networks.MAINNET && indexingConfig.db.toLowerCase().startsWith('sqlite')) {
+    logger.err(
+      'SQLite is not supported for indexing in production use. Please setup a Postgres instance and update the config file.'
```

Comment:
```suggestion
      'SQLite is not supported for the ComposeDB indexing database in production use. Please setup a Postgres instance and update the config file.'
```

Reply:
ComposeDB is the feature name people know (and included more than just indexing), so I'm trying to make sure we always talk about ComposeDB not just "indexing"

Reply:
Yeah that makes sense, I also standardized around `Compose DB`.